_tests/apps_test.go

```diff
@@ -1,26 +1,32 @@
 package _tests_test
 
 import (
+	"fmt"
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/config"
 	. "github.com/onsi/gomega"
 )
 
+func getRandAppName() string {
+	return fmt.Sprintf("apps-test-%d", GinkgoConfig.RandomSeed)
```

Comment:
won't this return the same value both times, since the seed is generated at execution time?